src/contracts/bapeliens-staking.json

```diff
@@ -0,0 +1,195 @@
546100ab906001600160a01b031681565b6040516001600160a01b0390911681526020015b60405180910390f35b6100db6100d6366004610551565b610163565b005b6100f06100eb366004610551565b6101dc565b6040516100bf9190610581565b6100db6102e4565b6100db6102f8565b6000546001600160a01b03166100ab565b61013161012c366004610551565b610377565b6040519081526020016100bf565b6001546001600160a01b03166100ab565b6100db61015e366004610551565b610388565b61016b6103f9565b6001600160a01b0381166101925760405163d92e233d60e01b815260040160405180910390fd5b600280546001600160a01b0319166001600160a01b0383169081179091556040517faec028e5a3ebac5c10179b9c24712afebe6baa8ff40ef73b162e3a54848356c690600090a250565b606060006101e983610453565b905060008167ffffffffffffffff811115610206576102066105c5565b60405190808252806020026020018201604052801561022f578160200160208202803683370190505b50905060005b828110156102dc5760025460405163e81c712760e01b81526001600160a01b038781166004830152602482018490529091169063e81c712790604401608060405180830381865afa15801561028e573d6000803e3d6000fd5b505050506040513d601f19601f820116820180604052508101906102b291906105db565b50508351849150839081106102c9576102c9610611565b6020908102919091010152600101610235565b509392505050565b6102ec6103f9565b6102f660006104ed565b565b60015433906001600160a01b0316811461036b5760405162461bcd60e51b815260206004820152602960248201527f4f776e61626c6532537465703a2063616c6c6572206973206e6f7420746865206044820152683732bb9037bbb732b960b91b60648201526084015b60405180910390fd5b610374816104ed565b50565b600061038282610453565b92915050565b6103906103f9565b600180546001600160a01b0383166001600160a01b031990911681179091556103c16000546001600160a01b031690565b6001600160a01b03167f38d16b8cac22d99fc7c124b9cd0de2d3fa1faef420bfe791d8c362d765e2270060405160405180910390a350565b6000546001600160a01b031633146102f65760405162461bcd60e51b815260206004820181905260248201527f4f776e61626c653a2063616c6c6572206973206e6f7420746865206f776e65726044820152606401610362565b6002546000906001600160a01b031661047f5760405163d92e233d60e01b815260040160405180910390fd5b6002546040516313ffffe960e11b81526001600160a01b038481166004830152909116906327ffffd290602401602060405180830381865afa1580156104c9573d6000803e3d6000fd5b505050506040513d601f19601f820116820180604052508101906103829190610627565b600180546001600160a01b031916905561037481600080546001600160a01b038381166001600160a01b0319831681178455604051919092169283917f8be0079c531659141344cd1fd0a4f28419497f9722a3daafe3b4186f6b6457e09190a35050565b60006020828403121561056357600080fd5b81356001600160a01b038116811461057a57600080fd5b9392505050565b6020808252825182820181905260009190848201906040850190845b818110156105b95783518352928401929184019160010161059d565b50909695505050505050565b634e487b7160e01b600052604160045260246000fd5b600080600080608085870312156105f157600080fd5b505082516020840151604085015160609095015191969095509092509050565b634e487b7160e01b600052603260045260246000fd5b60006020828403121561063957600080fd5b505191905056fea264697066735822122054f0e6dc9a0d3fff2f4d7d6dce3358d960c4e4f8d0c8477fa8195de5a523ab0164736f6c63430008130033",
```

Comment:
We can remove the `bytecode` and `deployedBytecode`.

Reply:
Resolved